texts/epn_paper.md

```diff
@@ -0,0 +1,74 @@
+If you do experiments, your lab is either closed or operates at a much lower capacity.
+Your travel plans are cancelled. Still, despite all this, you have likely attended a scientific presentation or even gave one yourself.
+The only difference is that it happened online.
+
+<!-- #### Online talks were gaining momentum already before, and for a good reason. -->
+
+Virtual conferences were gaining momentum even before the coronavirus crisis due to multiple advantages over traditional in-person gatherings.
+With the increasing concerns about greenhouse emissions, many prefer to limit their long-distance travel.
+For some community members with limited mobility, either for family, health, or budget reasons, traditional conferences are off limits altogether.
+Finally, most of the expensive and time-consuming aspects of conference organisation, such as finding a venue, lodging and catering for the participants are gone entirely.
+With all these obstacles out of the way, virtual conferences can be organized much faster and respond to the latest developments.
+They are also more inclusive and reach a broader audience especially if the talks are recorded.
+
+![](planning_checklist.png)
+
+**Figure caption:** A conference organizer checklist.
+
+<!-- #### We decided to develop online conferences and established VSF. -->
+
+The idea of organizing virtual conferences brought us together—a group of geographically and academically diverse physics researchers.
+We started exploring the opportunities and challenges of online conferences in late Summer 2019—well before the COVID-19 pandemic.
+We decided to keep our organization open for anyone to join, and all of our documentation and notes were public from the very beginning, with the thought that these should benefit anyone trying to work towards the same goal.
+It quickly became clear that we had the capacity to establish a venue for virtual conferencing. 
+So, we committed ourselves to developing this idea into a real platform, which we now call the [Virtual Science Forum](https://virtualscienceforum.org).
+
+<!-- #### In thinking about online events we had to revisit many assumptions about conferences. -->
+
+As we started to organize ourselves, we realized right away that in order to achieve our goals, we needed to revisit our assumptions about conferences.
+For example, the interactions between the speaker and the audience as well as among the audience themselves in an online event are quite different and much less spontaneous.
+The coffee break, such a natural part of in-person conferences, does not function effectively in an online event to spark discussions.
+Likewise, the length of an online event is constrained by very different factors, such as the spread of timezones, local commitments and distractions of the participants.
+
+<!-- #### We ran a couple of events, and saw that they work well. -->
+
+We created toolkits and developed strategies and guidelines to navigate these challenges.
```

Comment:
Not sure what we mean as "toolkits" especially in plural. What about something more specific:

```suggestion
To guide the community members, we wrote basic guidelines for all online event participants: organizers, moderators, and the audience.
```

Reply:
Should we maybe include a sentence somewhere earlier where we introduce the concept of a moderator?

Reply:
Indeed, "toolkit" implies to me that we developed some software or something similar. This change is more accurate to what you've done.

Reply:
@everthemore indeed, I still want to work on the paragraph about difference. It will benefit from extra detail.